packages/core/README.md

```diff
@@ -10,7 +10,7 @@ See https://github.com/Redocly/redocly-cli
 import { formatProblems, lint, loadConfig } from '@redocly/openapi-core';
 
 const pathToApi = 'openapi.yaml';
-const config = loadConfig('optional/path/to/.redocly.yaml');
+const config = loadConfig({ configPath: 'optional/path/to/.redocly.yaml' });
```

Comment:
This is a breaking change. This function can be used in other repositories/projects. We need to double-check it.

Reply:
I believe TS should help in other projects while updating `openapi-core`. We did such changes before.

Reply:
Hopefully no one else except us uses it yet 🤞 
Good.